Automattic/wp-calypso
# Fix/migrate enzyme to react testing library - Follow-button block

#### Changes proposed in this Pull Request

* The follow-button component is using the enzyme framework for testing, We need to migrate it to React-testing-library and jext

#### Testing instructions

1. Run the test for follow-button block with `yarn run test-client:watch ` from source directory
2. Select `P` and then type `follow-button`
3. We should see that the correct test file was executed and run like this:

![](https://d.pr/i/MlNluh+)
[Direct Link](https://d.pr/i/MlNluh+)



Related to #63194
